Minor improvements in the tutorial rcl pages (#3427)
ros2/ros2_documentation: source/Tutorials/Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace.rst

3 Clone a sample repo
^^^^^^^^^^^^^^^^^^^^^

Ensure you're still in the ``ros2_ws/src`` directory before you clone.

In the rest of the beginner developer tutorials, you will create your own packages, but for now you will practice putting a workspace together using existing packages.

The existing packages you will use are from the ``ros_tutorials`` repository (repo).

> If you went through the "Beginner: CLI Tools" tutorials, you'll be familiar with ``turtlesim``, one of the packages in this repo (see `on GitHub <https://github.com/ros/ros_tutorials/>`__).

Reviewer: ```suggestion
If you went through the :doc:`Beginner: CLI Tools <../../Beginner-CLI-Tools>` tutorials, you'll be familiar with ``turtlesim``, one of the packages in `ros_tutorials <https://github.com/ros/ros_tutorials/>`__.
```

(and then remove the sentence above saying `The existing packages you will use...`.
